composer.json

```diff
@@ -26,19 +26,19 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "7.4.*",
```

Comment:
not sure if we should already open up for php8

Reply:
Not unless we explicitly test for it